src/components/GetStarted.vue

```diff
@@ -4,7 +4,7 @@
     <div class="c-feature-box-title">Choose a Server</div>
     <div class="c-feature-box-para">
       Use 
-      <a href="#" class="is-link">our official server</a>
+      <span class="is-link">our official server</span>
```

Comment:
Any reason for this change? I think it's ok to keep this as an `a` tag. We just need to remember to add a link to it on launch

Reply:
Just thought the behaviour it results in (Jumping to the top of the page and appending # at the end of the url) was not really preferable. reverted it back.